source/deployment/bulk-loading-data-format.rst

```diff
@@ -14,7 +14,11 @@ Channel
 User
   Optional. If present, User objects must occur after the Team and Channel objects in the file and before any Post objects. Each User object defines the teams and channels that the user is a member of. If the corresponding teams and channels are not in the data file, then they must exist in the Mattermost database.
 Post
-  Optional. If present, Post objects must occur after all other objects in the file. Each Post object defines the team, the channel, and the username of the user who posted the message. If the corresponding team, channel, or user are not in the data file, then they must exist in the Mattermost database.
+  Optional. If present, Post objects must occur after the last User object but before any DirectChannel objects. Each Post object defines the team, the channel, and the username of the user who posted the message. If the corresponding team, channel, or user are not in the data file, then they must exist in the Mattermost database.
```

Comment:
I'll go with your preference but, as far as I'm aware, if "or" is used in a list - each thing listed is considered single - not part of a group.  However, if "and" is used, then the list of items becomes a group and can be plural.